pkg/actions/testdata/lua/catalogexport_hive_partition_pager.lua

```diff
@@ -0,0 +1,69 @@
+local hive = require("lakefs/catalogexport/hive")
+
+-- helper function to slice table array
+function table.slice(tbl, first, last, step)
```

Comment:
"table.slice" conflicts with standard [Lua table methods](https://www.lua.org/pil/19.1.html).  Perhaps pretend that these live in "xtable"?

Reply:
I think the link is wrong and I couldn't find any reference to what you're saying, can you link the source that shows it's a conflict with standard lua methods? 

Either way, I didn't like that fact that it's global table it's just a helper function for very specific use case that can't be shared outside this code anyway, so I changed the name and the scope.